content/_code-samples/checks/py/account_checks.py

```diff
@@ -0,0 +1,111 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountObjects
+from xrpl.utils import drops_to_xrp, ripple_time_to_datetime
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to the testnetwork
+
+
+
+"""Helper methods for working with token names"""
+
+def hex_to_symbol(hex: str = None) -> str:
+    """token hex_to_symbol."""
+    if len(hex) > 3:
+        bytes_string = bytes.fromhex(str(hex)).decode('utf-8')
+        return bytes_string.rstrip('\x00')
+    return hex
+
+
+# Query the ledger for all xrp checks an account has created or received
+
+# wallet address to query
+wallet_addr = str
```

Comment:
Similar to the comment on escrow code sample - https://github.com/XRPLF/xrpl-dev-portal/pull/1492#discussion_r964120827

Could you extend the example to take user input rather than an Object type there? 
Either a faucet wallet, or have a testnet account with some checks (sending and receiving) set up.